Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -287,6 +288,19 @@ class FakespotViewModel {
     let reviewQualityCardViewModel = FakespotReviewQualityCardViewModel()
     var optInCardViewModel = FakespotOptInCardViewModel()
 
+    var productAdViewModel = FakespotAdViewModel(productAdsData: ProductAdsData(
```

Comment:
Integration can be done with the API so we can avoid the mock data. I would suggest you removed this part that I assume was useful for testing.